integration-tests/actions.go

```diff
@@ -655,3 +655,84 @@ func (tr TestRun) redelegateTokens(action RedelegateTokensAction, verbose bool)
 		log.Fatal(err, "\n", string(bz))
 	}
 }
+
+type downtimeSlashAction struct {
+	chain     chainID
+	validator validatorID
+}
+
+func (tr TestRun) invokeDowntimeSlash(action downtimeSlashAction, verbose bool) {
+	// Bring validator down
+	tr.toggleValidatorDowntime(action.chain, action.validator, true, verbose)
+	// Wait appropriate amount of blocks for validator to be slashed
+	tr.waitBlocks(action.chain, 3, time.Minute)
+	// Bring validator back up
+	tr.toggleValidatorDowntime(action.chain, action.validator, false, verbose)
```

Comment:
Toggle usually implies an on/off switch (where you don't need to pass the parameter). But here a parameter is passed so another word might work better, like 'set', for example.

Reply:
Yea this makes sense to me 👍, the ideal calling syntax would be that the method makes a call to ```ip link``` to determine whether the node is currently up or down, then toggle the node to it's opposite state. However, I don't want to add more complexity/runtime to the method for small syntax gains. "Set" it is